src/pages/index.js

```diff
@@ -48,29 +52,81 @@ const cardData = {
 };
 
 function renderCard(cardData) {
-  const card = new Card(cardData, "#card-template", (name, link) => {
-    viewImagePopup.open({ name, link });
+  // const card = new Card(cardData, "#card-template", (name, link) => {
```

Comment:
Needs correction: please check the functionality for adding new cards. when i add a card it has no data

Reply:
hi! can you clarify on what you mean by no data please? i see data there when i add a card. please see screenshot. thank you in advance! 
![new card info](https://github.com/kimyeeeee/se_project_aroundtheus/assets/122238105/e4e43e0b-1775-4db9-a31c-e86f4f3a8d53)